leankyr/Easy_Problems/Game_Of_Thrones/gameOfThrones.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3.5
+# -*- coding: utf-8 -*-
+
+import math
+import os
+import random
+import re
+import sys
+
+
+
+def gameOfThrones(s):
+    s = s.rstrip('\n')
+    counter = 0
+    lettermap = dict(zip("abcdefghijklmnopqrstuvwyz",range(0,26)))
+    times_found = [0]*26
+
+    for c in s:
```

Comment:
What you do here can be done easily with the [Counter data structure](https://docs.python.org/3/library/collections.html#collections.Counter) and you don't need to create the lettermap